google/cloud/logging_v2/handlers/handlers.py

```diff
@@ -225,6 +232,20 @@ def emit(self, record):
             source_location=record._source_location,
         )
 
+    def flush(self):
+        """Forces the Transport object to submit any pending log records.
+
+        For SyncTransport, this is a no-op.
+        """
+        super(CloudLoggingHandler, self).flush()
+        if self.transport is not None:
+            self.transport.flush()
+
+    def close(self):
+        """Closes the log handler and cleans up all Transport objects used."""
+        self.transport.close()
+        self.transport = None
```

Comment:
I don't like the idea of using `transport == None` as a way to check if the handler is closed. That makes the types quite a bit more complicated.

Can't we use a new flag for this?